plugins/fastboot/fu-fastboot-device.c

```diff
@@ -318,8 +318,10 @@ fu_fastboot_device_setup (FuDevice *device, GError **error)
 	/* bootloader version */
 	if (!fu_fastboot_device_getvar (device, "version-bootloader", &version_bootloader, error))
 		return FALSE;
-	if (version_bootloader != NULL && version_bootloader[0] != '\0')
-		fu_device_set_version_bootloader (device, version_bootloader, FWUPD_VERSION_FORMAT_PAIR);
+	if (version_bootloader != NULL && version_bootloader[0] != '\0') {
+		fu_device_set_version_format (device, FWUPD_VERSION_FORMAT_PAIR);
```

Comment:
I'm not sure this is actually needed now - it inherits from USB no?

Reply:
Usb is BCD I think